actionmailer/test/message_delivery_test.rb

```diff
@@ -76,7 +76,7 @@ def test_should_enqueue_and_run_correctly_in_activejob
 
   test "should enqueue a delivery with a delay" do
     travel_to Time.new(2004, 11, 24, 01, 04, 44) do
-      assert_performed_with(job: ActionMailer::DeliveryJob, at: Time.current.to_f + 600.seconds, args: ["DelayedMailer", "test_message", "deliver_now", 1, 2, 3]) do
+      assert_performed_with(job: ActionMailer::DeliveryJob, at: Time.current.to_f + 600, args: ["DelayedMailer", "test_message", "deliver_now", 1, 2, 3]) do
```

Comment:
@dhh this has now been changed to `Time.current + 600.seconds` which it what it should've been in the first place. Also changed another test to `Time.current + 1.hour` instead of `Time.now.to_f + 3600`.